rust/stackable-cockpit/src/xfer/mod.rs

```diff
@@ -19,8 +19,11 @@ type Result<T, E = Error> = core::result::Result<T, E>;
 
 #[derive(Debug, Snafu)]
 pub enum Error {
-    #[snafu(display("failed to read local file"))]
-    ReadLocalFile { source: std::io::Error },
+    #[snafu(display("failed to read from local file {file:?}"))]
+    ReadLocalFile {
+        source: std::io::Error,
+        file: PathBuf,
+    },
```

Comment:
suggestion: Use `path` as the field name instead.

```suggestion
    #[snafu(display("failed to read from local file from {path:?}"))]
    ReadLocalFile {
        source: std::io::Error,
        path: PathBuf,
    },
```

Reply:
I intentionally choose file over path but no strong opinion, picking yours ;)